src/main/scala/com/databricks/spark/csv/DefaultSource.scala

```diff
@@ -49,7 +50,7 @@ class DefaultSource
       parameters: Map[String, String],
       schema: StructType) = {
     val path = checkPath(parameters)
-    val delimiter = parameters.getOrElse("delimiter", ",")
+    val delimiter = StringEscapeUtils.unescapeJava(parameters.getOrElse("delimiter", ","))
```

Comment:
can't we just replace "\t" string literal with a tab here? I know there are other things, but I don't think we should handle everything anyway (like \n)